mhn/api/views.py

```diff
@@ -249,6 +250,69 @@ def top_attackers():
     )
 
 
+def get_credentials_payloads(clio, limit=10000, hours_ago=4):
+    credentials_payloads = []
+    credentials_payloads += clio.hpfeed.get_payloads({'limit': limit}, {"channel": "cowrie.sessions",
+                                                                        "timestamp":
+                                                                        {'$gte': datetime.datetime.now() -
+                                                                         datetime.timedelta(hours=hours_ago)}})[2]
+    return credentials_payloads
+
+
+@api.route('/credentials.csv/', methods=['GET'])
+@token_auth
+def credential_list_csv():
+    fieldnames = ['username', 'password', 'count']
+
+    options = request.args.to_dict()
+    limit = int(options.get('limit', '1000'))
```

Comment:
Limit set to 1,000 here...are these the same limits? Should they be set the same if so?

Reply:
Doesn't really matter. Not necessarily, because that function may be reused for other API calls in the future. It gets overridden. That being said, I can change it for consistency's sake